vsts/pipelines/templates/_integrationJobTemplate.yml

```diff
@@ -114,4 +114,22 @@ jobs:
       echo "##vso[task.setvariable variable=PushRuntimeImages;]false"
       echo "##vso[task.setvariable variable=EmbedBuildContextInImages;]false"
     displayName: 'Set variables'
-  - template: _buildTemplate.yml
\ No newline at end of file
+  - template: _buildTemplate.yml
+
+- job: Job_NotifyFailure
+  displayName: Create Bug if Build Fails
+  dependsOn: 
```

Comment:
Shouldn't we also have a condition to run this job only when any jobs this depends on fails?